BeagleBone_black/src/state_machine/main.cpp

```diff
@@ -51,7 +51,9 @@ void Main::run()
   while (1) {
     checkFailure();
     checkCommunications();
-    checkNavigation();
+    if (sm_data_.current_state != data::kIdle) {
```

Comment:
@mkristien This check already ensures that the module status is not kStart when checking the navigation data.